Objects/listobject.c

```diff
@@ -85,9 +85,6 @@ list_resize(PyListObject *self, Py_ssize_t newsize)
     }
     else {
         // integer overflow
-        items = NULL;
```

Comment:
What it 'items = (PyObject **)PyMem_Realloc(self->ob_item, num_allocated_bytes);' is failed?

Reply:
Thank you for the prompt feedback. It was something I had not considered.

Thanks to you, my curiosity has been satisfied. Thank you.